data/seeds/001-projects.js

```diff
@@ -0,0 +1,9 @@
+exports.seed = function(knex, Promise) {
```

Comment:
Not sure if this worked out for you since you're missing the incremental primary keys. Should be:

exports.seed = function(knex, Promise) {
  return knex("projects").insert([
    { id: 1, name: "Clean garage", description: "sweep the floor, move stuff out of the way, boom", completed: "false" },
    { id: 2, name: "test", description: "try to add more than one seed field", completed: "false" }
  ]);
};